apps/admin_app/lib/admin_app_web/templates/product/form.html.eex

```diff
@@ -10,7 +10,7 @@
       <a class="nav-link active" href="#tab1default" data-toggle="tab">Product Detail</a>
    </li>
    <%= if  @conn.request_path != product_path(@conn, :new) do %>
-   <%= if !is_child_product(@parent_product) do %>
+   <%= if !Product.is_child_product(@parent_product) do %>
```

Comment:
@jyotigautam use `unless` here please.

Reply:
@jyotigautam @ashish173 
I would say call it from helper. Calling module name in views will make it difficult to change.

Like what if someone wants to rename `is_child_product` later. Or want to modify its `arity` later. Then all the views will be needed to be updated.

Instead just call the function from a view helper.